devito/finite_differences/differentiable.py

```diff
@@ -303,12 +303,12 @@ def shift(self, dim, shift):
         return self._subs(dim, dim + shift)
 
     @property
-    def laplace(self):
+    def laplace(self, **kwargs):
```

Comment:
No laplace is a propery and doesn't take kwargs, laplacian is the one with the kwargs